contract/deploy/ensDeployer.ts

```diff
@@ -61,14 +62,13 @@ const f = async function (hre: HardhatRuntimeEnvironment) {
     BaseRegistrarImplementation: await ensDeployer.baseRegistrar(),
     FIFSRegistrar:  await ensDeployer.fifsRegistrar(),
     ReverseRegistrar: await ensDeployer.reverseRegistrar(),
-    BaseRegistrar: await ensDeployer.baseRegistrar(),
     MetadataService: await ensDeployer.metadataService(),
     NameWrapper: await ensDeployer.nameWrapper(),
     ETHRegistrarController: await ensDeployer.registrarController(),
     PublicResolver: await ensDeployer.publicResolver(),
     UniversalResolver: await ensDeployer.universalResolver(),
     Multicall: await Multicall.address,
-  }) + '\'')
+  } , null, 2) + '\'')
```

Comment:
we actually need this to be in a single line, to be used in ens-app-v3 env

Reply:
I tested multiline .env.local in ens-app-v3 and it seems to parse it ok.